### ISSUE TITLE
Spacewalk bridge - historical times of each transfer in the transaction history are not constant

### ISSUE THREAD

**Issue by annatekl** [pendulum-chain/portal#447]
### Describe the bug

Historical times of each transfer in the transaction history are not constant. The times are changing constantly (every 10 seconds) by a few seconds/minutes. Looks like they are jumping randomly.

### Reproduction

1. Go to Spacewalk bridge
2. Perform bridge to Pendulum or back to Stellar
3. Go to transfers and check the timestamp
![Screenshot 2024-05-14 at 19 29 48](https://github.com/pendulum-chain/portal/assets/138758794/f5bce179-3700-4526-af4b-171765a1252a)
![Screenshot 2024-05-14 at 19 30 14](https://github.com/pendulum-chain/portal/assets/138758794/d2310e14-8c5d-4246-b422-267e3419e9b1)

### Expected behavior

Constant timestamp for each of the statuses.

**Comment by annatekl** [comment]
ticket updated @ebma

**Comment by prayagd** [comment]
@prayagd to add the suggestion to add a subscan link to the transfers

**Comment by ebma** [comment]
Shouldn't the subscan link be part of a different ticket though? Also note that this ticket is blocked by https://github.com/pendulum-chain/pendulum-squids/issues/66.

**Comment by ebma** [comment]
How come this ticket is so high up the priority list but the related ticket for the indexer is not even in the ready column yet? This ticket cannot be worked on before the changes to the indexer were added.

**Comment by ebma** [comment]
I don't think this ticket needs further refinement at the moment. I just wanted to make clear that this is blocked by the linked indexer ticket.

**Comment by TorstenStueber** [comment]
@pendulum-chain/product why is this not in Ready?

**Comment by prayagd** [comment]
Can be moved to ready now
